.kokoro/continuous/samples.cfg

```diff
@@ -16,10 +16,15 @@ gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
 # Use the trampoline script to run in docker.
 build_file: "google-cloud-ruby/.kokoro/trampoline_v2.sh"
 
-# Configure the docker image for kokoro-trampoline.
 env_vars: {
   key: "TRAMPOLINE_IMAGE"
-  value: "gcr.io/cloud-devrel-kokoro-resources/yoshi-ruby/release"
+  value: "gcr.io/cloud-devrel-kokoro-resources/yoshi-ruby/sample-acceptance"
```

Comment:
Instead of creating your own Dockerfile, we should just add any needed installations to the existing Dockerfile in the testing-infra-docker repository and rebuild those (which will happen automatically after changes are merged in that repo, though the build process can take a couple hours). We're using a single Dockerfile (the one in testing-infra-docker) for all Ruby's Kokoro jobs so we don't have to maintain multiple Dockerfiles. And also so we don't spend more minutes building another docker image every time we run the testing job.

So don't change this value, and don't add a TRAMPOLINE_DOCKERFILE environment variable, and don't add a Dockerfile to this repo. Just append your changes to the existing one.

Reply:
created a PR to update the existing image https://github.com/googleapis/testing-infra-docker/pull/184